web/src/lib/utils/asset-utils.ts

```diff
@@ -38,20 +38,23 @@ export const addAssetsToAlbum = async (albumId: string, assetIds: string[]) => {
   });
   const count = result.filter(({ success }) => success).length;
   const $t = get(t);
-  notificationController.show({
-    type: NotificationType.Info,
-    timeout: 5000,
-    message:
-      count > 0
-        ? $t('assets_added_to_album_count', { values: { count: count } })
-        : $t('assets_were_part_of_album_count', { values: { count: assetIds.length } }),
-    button: {
-      text: $t('view_album'),
-      onClick() {
-        return goto(`${AppRoute.ALBUMS}/${albumId}`);
+
+  if (showNotification) {
```

Comment:
maybe  doing if(!showNotifications) { return }; would be better (everything else stays not indented)